pymc/gp/cov.py

```diff
@@ -129,6 +141,10 @@ def __array_wrap__(self, result):
                 "Known types are `Add` or `Prod`."
             )
 
+    @staticmethod
+    def _alloc(X, *shape: int) -> TensorVariable:
+        return pt.alloc(X, *shape)  # type: ignore
+
 
```

Comment:
Is this factored out for mypys sake?

Reply:
Ah, yes, this was for mypys sake...I think because `pt.alloc` can technically return things other than `TensorVariable`...

Reply:
It cannot, but mypy doesn't understand that. If you would like ta take a look at the PyTensor level to see if it can be fixed that would be great!

Reply:
Ah yeah - let me take a look